server/services/calculateReleaseDatesService.ts

```diff
@@ -189,89 +195,6 @@ export default class CalculateReleaseDatesService {
     return new CalculateReleaseDatesApiClient(token).getCalculationBreakdown(calculationRequestId)
   }
 
-  // Find which sentence provides effective dates.
```

Comment:
This code was related to the derivation of release dates using effective dates - not required anu more because we get the breakdown details directly from the API

Reply:
Good work! I hated this code